package.json

```diff
@@ -18,9 +18,9 @@
     "build": "gulp build",
     "build:browser": "gulp build:browser",
     "prepublish": "npm run lint && npm run test && npm run compile",
-    "lint": "eslint 'src/**/*.js' 'test/*.js'",
+    "lint": "eslint \"src/**/*.js\" \"test/*.js\"",
     "lint:fix": "npm run lint -- --fix",
-    "test": "nyc --reporter=html --reporter=text mocha test/setup.js --sort 'src/**/*.spec.js' --compilers js:babel-core/register --timeout 30000",
+    "test": "nyc --reporter=html --reporter=text mocha test/setup.js --sort \"src/**/*.spec.js\" --compilers js:babel-core/register --timeout 30000",
```

Comment:
please, make sure this doesn't come as new change, since it was introduced by https://github.com/gerardobort/node-corenlp/pull/51 ... rebasing your branch with the latest master HEAD should do the trick

Reply:
Wow, thats great. I never had a concrete use-case for git rebase and therefore probably never really understood it. Thanks for changing that. 😄